packages/lesswrong/components/recommendations/PostBottomRecommendations.tsx

```diff
@@ -0,0 +1,163 @@
+import React from "react";
+import { Components, registerComponent } from "../../lib/vulcan-lib";
+import { Link } from "../../lib/reactRouterWrapper";
+import { userGetProfileUrl } from "../../lib/collections/users/helpers";
+import { useRecentOpportunities } from "../hooks/useRecentOpportunities";
+import { AnalyticsContext } from "../../lib/analyticsEvents";
+import { useMulti } from "../../lib/crud/withMulti";
+import { usePaginatedResolver } from "../hooks/usePaginatedResolver";
+import { MAX_CONTENT_WIDTH } from "../posts/TableOfContents/ToCColumn";
+
+const styles = (theme: ThemeType) => ({
+  root: {
+    background: theme.palette.grey[55],
+    padding: "80px 0",
+    marginTop: 80,
+  },
+  section: {
+    maxWidth: MAX_CONTENT_WIDTH,
+    margin: "0 auto 60px",
+  },
+  sectionHeading: {
+    fontFamily: theme.palette.fonts.sansSerifStack,
+    fontSize: 20,
+    fontWeight: 600,
+    color: theme.palette.grey[1000],
+    marginBottom: 16,
+  },
+  largePostItem: {
+    marginTop: 8,
+  },
+  viewMore: {
+    marginTop: 12,
+    fontFamily: theme.palette.fonts.sansSerifStack,
+    fontSize: 14,
+    fontWeight: 600,
+    color: theme.palette.grey[600],
+  },
+});
+
+const PostBottomRecommendations = ({post, classes}: {
+  post: PostsWithNavigation | PostsWithNavigationAndRevision,
+  classes: ClassesType,
+}) => {
+  const {
+    results: moreFromAuthorPosts,
+    loading: moreFromAuthorLoading,
+  } = useMulti({
+    collectionName: "Posts",
+    fragmentName: "PostsListWithVotes",
+    terms: {
+      userId: post.userId,
+      sortedBy: "topAdjusted",
+      limit: 3,
+    },
+  });
+
+  const {
+    results: digestPosts,
+    loading: digestLoading,
+  } = usePaginatedResolver({
+    fragmentName: "PostsPage",
+    resolverName: "DigestPostsThisWeek",
+    limit: 3,
+  });
+
+  const {
+    results: opportunityPosts,
+    loading: opportunitiesLoading,
+  } = useRecentOpportunities({
+    fragmentName: "PostsListWithVotes",
+  });
+
+  const profileUrl = userGetProfileUrl(post.user);
+
+  const hasUserPosts = post.user &&
+    (moreFromAuthorLoading || !!moreFromAuthorPosts?.length);
+
+
+  const {
+    PostsLoading, ToCColumn, EAPostsItem, EALargePostsItem, UserTooltip,
+  } = Components;
+  return (
+    <AnalyticsContext pageSectionContext="postPageFooterRecommendations">
+      <div className={classes.root}>
+        <ToCColumn tableOfContents={<div />}>
```

Comment:
Adding the noop table of contents appears to have caused this thing to appear in the top left:
<img width="537" alt="Screenshot 2023-09-25 at 10 44 44" src="https://github.com/ForumMagnum/ForumMagnum/assets/77623106/309c4992-6d57-4f37-906a-46e2f694ebe9">
<img width="537" alt="Screenshot 2023-09-25 at 10 44 48" src="https://github.com/ForumMagnum/ForumMagnum/assets/77623106/eea90383-163d-43e1-84b3-d3c6095b4080">